cohere/responses/custom_model.py

```diff
@@ -104,6 +106,7 @@ def from_dict(cls, data: Dict[str, Any]) -> "CustomModel":
             model_type=REVERSE_CUSTOM_MODEL_PRODUCT_MAPPING[data["settings"]["finetuneType"]],
             created_at=_parse_date(data["created_at"]),
             completed_at=_parse_date(data["completed_at"]) if "completed_at" in data else None,
+            base_model=data["settings"]["baseModel"],
```

Comment:
is `baseModel` correct? shouldn't it be `base_model` https://github.com/cohere-ai/blobheart/blob/main/go/pkg/data/finetune.go#L33

Reply:
Yes I tried it again, and it's `baseModel` and not `base_model`. This is consistent with the line 106 above with `data["settings"]["finetuneType"]` instead of `data["settings"]["finetune_type"]`

Reply:
weird. do you know where these values are coming from? we should try to make it consistent if possible, otherwise i anticipate bugs

Reply:
The class to look at is that one: https://github.com/cohere-ai/blobheart/blob/alexandre/add_basemodel_rerank/proto/blobheart.gen.go#L1546

Reply:
ooof. this is very inconsistent. ok let's keep it as is for now, and also bring this up to the OS team